src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1302,15 +1322,7 @@ public static File getRevisionFile(Run build) {
         return new File(build.getRootDir(),"revision.txt");
     }
 
-    /**
-     * @deprecated use {@link hudson.scm.SubversionSCM#getRevisionFile(hudson.model.Run)} instead.
-     *
-     * Gets the file that stores the revision.
-     */
-    @Deprecated
-    public static File getRevisionFile(AbstractBuild build) {
```

Comment:
This change is unrelated to the pull request title.
IMO it should be reverted, because it breaks the backward compatibility.